content/(bot)/leveling-commands/background/delete.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Delete Background
+description: Delete custom rank card background.
+---
+
+# Delete
+### Description
+
+<Callout type="warning">
+This is a **method** or **sub-command** of the [Background ](./)command. It is not its own command.
+</Callout>
+
+This method of the [background ](./)command can be used to delete a set leveling background.
+
+You cannot delete a custom leveling background if one has not been set yet. Deleting your custom leveling background will revert your leveling card back to the default 'Wavy Grey' background.
+
+Deleting your custom background will delete it forever instantly. There is no way to recover the image.
+
+If you intend to keep your background for a later date, consider using the [view method](view.md) to save it.
+
+### Command Structure
+
+```
+/background delete
+```
+
+<figure><img src="https://i.imgur.com/jNYrQXL.png" alt=""><figcaption></figcaption></figure>
```

Comment:
All these kind of images should use md syntax for images instead. e.g.
`![](https://...)`